cli/tests/integration/coverage_tests.rs

```diff
@@ -504,3 +504,72 @@ fn no_internal_node_code() {
     assert_starts_with!(url, "file:");
   }
 }
+
+#[test]
+fn test_html_reporter() {
+  let context = TestContext::default();
+  let tempdir = context.temp_dir();
+  let tempdir = tempdir.path().join("cov");
+
+  let output = context
+    .new_command()
+    .args_vec(vec![
+      "test".to_string(),
+      "--quiet".to_string(),
+      format!("--coverage={}", tempdir),
+      "coverage/multisource".to_string(),
+    ])
+    .run();
+
+  output.assert_exit_code(0);
+  output.skip_output_check();
+
+  let output = context
+    .new_command()
+    .args_vec(vec![
+      "coverage".to_string(),
+      "--html".to_string(),
+      format!("{}/", tempdir),
+    ])
+    .run();
+
+  output.assert_exit_code(0);
+  output.assert_matches_text("HTML coverage report has been generated at [WILDCARD]/cov/html/index.html\n");
```

Comment:
This will need to be tweaked depending on *nix/Windows

Reply:
Updated. Now it uses urls for the path